src/webapi/element.rs

```diff
@@ -163,8 +163,46 @@ pub trait IElement: INode + IParentNode + IChildNode {
     fn has_pointer_capture( &self, pointer_id: i32 ) -> bool {
         js!( return @{self.as_ref()}.hasPointerCapture( @{pointer_id} ); ).try_into().unwrap()
     }
+
+    /// Insert nodes from HTML fragment into specified position.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML)
+    fn insert_adjacent_html( &self, position: &str, html: &str ) -> Result<(), SyntaxError> {
```

Comment:
1. The `position` parameter should be an enum.
2. This can also throw `NoModificationAllowedError` besides `SyntaxError`. (See `WebSocket::new` for an example on how to handle this. We don't have bindings for `NoModificationAllowedError` so you have to add that too.)
3. Please add a link to the specs' IDL just as we have for the most of other methods.

Reply:
1. Done
2. Done.
3. Done.